src/components/Player/Pages/Histograms/Histograms.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Graph } from 'components/Visualizations';
+import { getPlayerHistogram } from 'actions';
+import { playerHistogram } from 'reducers';
+import { withRouter } from 'react-router';
+import FlatButton from 'material-ui/FlatButton';
+import { deSnake } from 'utility';
+import histogramNames from './histogramNames';
+import styles from './Histograms.css';
+
+
+const getAxis = (name, show = true) => ({
+  label: {
+    text: name,
+    show,
+  },
+});
+
+const selectHistogram = (router, histogramName, playerId) => {
+  router.push(`/players/${playerId}/histograms/${histogramName}`);
+};
+
+const Histogram = ({ histogramName, columns, router, playerId }) => (
```

Comment:
I can do that, but then it won't be routed to the right place in the URL (it will be /histograms), and then if I do the trick I mentioned above then it will break the back button.